src/AzureStack/Admin/ComputeAdmin/Compute.Admin.Tests/src/DiskMigrationJobTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿
```

Comment:
Can you do tests that should fail?  For instance try to migrate to a share that doesn't exist or with a disk that is invalid?

Reply:
Try `Assert.False` or `Assert.Exception` constructs.
If you wish to add the tests but enable them at a later time, you could always skip the tests, hope this helps

Reply:
I have added more tests.